### ISSUE TITLE
Unit tests failing in nightly builds

### ISSUE THREAD

**Issue by SagarGi** [owncloud/update-testing#41]
Drone nightly build https://drone.owncloud.com/owncloud/update-testing/1612/1/6

```
console
7.4: Pulling from owncloudci/php
Digest: sha256:94d945c4eb485a1db0b8323e66bfef86fdfc778107384712f686856896a5c830
Status: Image is up to date for owncloudci/php:7.4
+ cd owncloud
+ php ./occ app:disable gallery
No such app enabled: gallery
+ chmod +x lib/composer/phpunit/phpunit/phpunit
+ cd tests
+ ../lib/composer/phpunit/phpunit/phpunit --configuration phpunit-autotest.xml --log-junit "autotest-results-$DATABASE.xml"
PHP Fatal error:  Uncaught Error: Class 'OCA\Notifications\Tests\Unit\TestCase' not found in /drone/src/owncloud/apps/notifications/tests/unit/AppInfo/AppTest.php:34
Stack trace:
#0 /drone/src/owncloud/tests/apps.php(23): require_once()
#1 /drone/src/owncloud/tests/apps.php(21): loadDirectory()
#2 /drone/src/owncloud/tests/apps.php(21): loadDirectory()
#3 /drone/src/owncloud/tests/apps.php(54): loadDirectory()
```

Unit tests for update-testing app fails on nightly on every matrix piprline on the above. This needs to be inviestigated as it fails on every nightly builds.

**Comment by phil-davis** [comment]
It is likely that this is related to https://github.com/owncloud/notifications/pull/363

The name change from `tests/Unit` to `tests/unit` may have broken something here.

**Comment by phil-davis** [comment]
Fixed by #42 - please reopen if it fails again tonight.
